app/styles/btn.less

```diff
@@ -69,16 +93,37 @@
     padding-top: 0;
 }
 
+
+
 .theme-1 .btn, .theme-1.btn {
     background-color: @color-theme-1-primary-bg;
+    border-color: @color-theme-1-primary-bg;
     color: @color-theme-1-primary-fg;
 }
+.theme-1-hollow .btn, .theme-1-hollow.btn {
```

Comment:
It's arguably a matter of preference, but, since the intent of outline buttons is functional more than stylistic (no way @jcarroll2007 could have known, btw), it's probably more BEM-conformant to define a separate class `btn--outline` and continue using the existing themes. I'll rework this after the merge.